cns/restserver/ipam.go

```diff
@@ -274,6 +292,7 @@ func filterIPConfigMap(toBeAdded map[string]cns.IPConfigurationStatus, f func(cn
 
 //SetIPConfigAsAllocated takes a lock of the service, and sets the ipconfig in the CNS state as allocated, does not take a lock
 func (service *HTTPRestService) setIPConfigAsAllocated(ipconfig cns.IPConfigurationStatus, podInfo cns.KubernetesPodInfo, marshalledOrchestratorContext json.RawMessage) cns.IPConfigurationStatus {
+	logger.Printf("[setIPConfigAsAllocated] Marking IP [%v] to Allocated for Pod [%v]", ipconfig, podInfo)
```

Comment:
similar suggestion as above, if you are going to print the entire structs, use `%+v`. Same for all similar cases.